packages/contracts/deploy-config/goerli-qa.ts

```diff
@@ -1,16 +1,15 @@
 const config = {
   numDeployConfirmations: 1,
-  l1BlockTimeSeconds: 15,
-  l2BlockGasLimit: 15_000_000,
-  l2ChainId: 1705003,
-  updateGaslimitBlock: 90000,
-  eigenDaBlock: 2512521,
+  gasPrice: 150_000_000_000,
+  l1BlockTimeSeconds: 12,
+  l2BlockGasLimit: 30_000_000,
+  l2ChainId: 5000,
```

Comment:
`L2ChainID` of QA network based on Goerli looks wrong.

Reply:
done